src/cpp/fastdds/builtin/type_lookup_service/TypeLookupManager.cpp

```diff
@@ -224,13 +224,12 @@ void TypeLookupManager::remove_remote_endpoints(
 }
 
 fastrtps::rtps::SampleIdentity TypeLookupManager::get_type_dependencies(
-        const fastrtps::types::TypeIdentifierSeq& id_seq) const
+        const xtypes::TypeIdentifierSeq& id_seq) const
 {
     fastrtps::rtps::SampleIdentity id = INVALID_SAMPLE_IDENTITY;
 
     TypeLookup_getTypeDependencies_In in;
-    //TODO with xtypes::TypeIdentifierSeq
-    // in.type_ids() = id_seq;
+    in.type_ids() = id_seq;
     (void) id_seq;
```

Comment:
This casting is no longer required. Please remove here and in any other similar ocurrence.

Reply:
Fixed in [523b3e4](https://github.com/eProsima/Fast-DDS/pull/4126/commits/523b3e44f46779bec49ea92cea752085b202fb56)